x/marketmap/keeper/keeper.go

```diff
@@ -0,0 +1,116 @@
+package keeper
+
+import (
+	"cosmossdk.io/collections"
+	"cosmossdk.io/core/store"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	slinkytypes "github.com/skip-mev/slinky/pkg/types"
+	"github.com/skip-mev/slinky/x/marketmap/types"
+)
+
+type Keeper struct {
+	cdc codec.BinaryCodec
+
+	// keeper dependencies
+	oracleKeeper types.OracleKeeper
```

Comment:
I think we really should not be adding anything from `x/oracle` as a dependency into `x/marketmap`.  My thinking was for `x/marketmap` to expose [hooks](https://github.com/cosmos/cosmos-sdk/blob/main/x/staking/types/hooks.go#L121) in a pattern similar to how some other cosmos sdk modules do. 

Something like `AfterMarketMapUpdate()` which can be used by `x/oracle` (or `x/prices`) to get a view of changes without adding a direct keeper dependency

Reply:
Could get around this via an interface + implementation for the x/oracle module?

Reply:
Removing oracleKeeper interface from deps here. Plan is to have a follow up PR which includes hooks.